erpnext_com/www/docs/user/manual/en/loan-management/loan-security-shortfall.md

```diff
@@ -0,0 +1,34 @@
+<!-- add-breadcrumbs -->
+# Loan Security Shortfall
+> Introduced in Version 13
+
+**If loan to security value ratio falls below a specific value a Loan Security Shortfall is automatically created for that specific loan.**
+
+<img class="screenshot" alt="Loan Security Shortfall" src="{{docs_base_url}}/assets/img/loan-management/loan-security-shortfall-flow.png">
+
+To access the Loan Security Shortfall list, go to:
+> Home > Loan Management > Loan Security > Loan Security Shortfall
+
+<img class="screenshot" alt="Loan Application" src="{{docs_base_url}}/assets/img/loan-management/loan-security-shortfall.png">
+
+
+A Loan Security Shortfall contains the details about:
+
+  * The Loan against which there is a Loan to security value shortfall
```

Comment:
format like 
**Loan Amount**: The amount in the Loan.
or
arrange as bullets if explanation is trivial